_posts/2025-01-29-cockpit-332.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Cockpit 332
+author: jelle
+date: '2025-01-29'
+tags: cockpit, machines, podman, files
+slug: cockpit-332
+category: release
+summary: 'package-less session support for more operation systems, show system boot times in metrics, automatically start podman.socket and improve quadlet support in cockpit-podman, show user group in files footer'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 332, cockpit-machines 327, cockpit-podman 100, and cockpit-files 15:
+
+## cockpit/ws container: Include cockpit-files
+
+The [cockpit/ws container](https://quay.io/repository/cockpit/ws) now includes [cockpit-files](https://github.com/cockpit-project/cockpit-files/). When you log into a remote machine that does not have any cockpit packages installed (package-less mode), the "Files" page is now available.
```

Comment:
```suggestion
The [cockpit/ws container](https://quay.io/repository/cockpit/ws) now includes [cockpit-files](https://github.com/cockpit-project/cockpit-files/). When you log into a remote machine that does not have any Cockpit packages installed (package-less mode), the "Files" page is now available.
```

This needs to either be "Cockpit" or `cockpit` here... either it's Cockpit packages (as in packages related to Cockpit) or `cockpit` packages (as in packages that contain the name `cockpit`).